blog/.vuepress/components/preview.vue

```diff
@@ -1,21 +1,21 @@
 <template>
   <div class="Preview">
     <div class="Preview-left">
-      <img class="Preview-img" :src="meetup.old.backgroundImg" alt="vue montreal" />
```

Comment:
I'm not sure why the key was named `old`, I removed it since now `meetup` is the `event` object